apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/BFD-766-fix-beneficiary-monthly-data/Playbook.txt

```diff
@@ -145,7 +138,7 @@ action(s) need to be taken.
 
 Q. What happens if Step 6 fails?
 
-   Step 6 can be run miultiple times; it is effectively running an idempotent UPDATE so
+   Step 6 can be run multiple times; it is effectively running an idempotent UPDATE so
    re-running would simply update data with the same data as a previous run.
```

Comment:
It's technically not idempotent anymore.
```suggestion
   Step 6 can be run multiple times; it will pick back up where it left off.
```

Reply:
Good point...we'd have to reload the CCW temp table with the csv file(s)..